packages/iov-bns/src/bnsconnection.spec.ts

```diff
@@ -388,7 +388,7 @@ describe("BnsConnection", () => {
       // expect(headers[0].id).not.toEqual(headerFromGet.id);
       expect(headers[0].transactionCount).toBeGreaterThanOrEqual(0);
       expect(headers[0].time.getTime()).toBeGreaterThan(headerFromGet.time.getTime() - 1200);
-      expect(headers[0].time.getTime()).toBeLessThan(headerFromGet.time.getTime() + 1200);
+      expect(headers[0].time.getTime()).toBeLessThan(headerFromGet.time.getTime() - 800);
```

Comment:
Nice catch! Could you pull out the 1000 into a `blockTime` variable to make clear what this is supposed to mean?